UN-OCHA/unocha-site
# [UNO-757] Featured stories

Refs: UNO-757

This adds a featured stories to the `stories` paragraph type to allow editors to explicitly select featured stories for a particular page to have better control instead of using the sticky/promote.

<img width="920" alt="Screenshot 2023-08-03 at 8 50 02" src="https://github.com/UN-OCHA/unocha-site/assets/696348/d539cffb-7415-4d4f-b184-65d3c83808ca">

## Tests

1.  Checkout the branch, clear the cache, import the config
2. Edit the homepage, edit the stories paragraph type, select some stories to feature
3. Save and confirm that the selected stories appear first and in order and with the "featured" label, and the special styling for the first one
4. Repeat (3) and (4) for a region, it should be similar
5. Repeat (3) and (4) for a response, it should be similar without the special styling for the first item and with only 3 stories.
